docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -79,11 +80,41 @@ Here are the steps to configure a new Google Service Account Key in EAS for send
 </Step>
 
 <Step label="5">
+  Configure the `google-services.json` file in your app. (Note: you may skip this step if `google-services.json` has already been set up.)
+
+  First, download the `google-services.json` file from Firebase Cloud Console and place it somewhere in your app directory (the root is fine). 
```

Comment:
```suggestion
  Configure the **google-services.json** file in your project. Download it from the Firebase Console and place it at the root of your project directory.
  
  > **Note**: You can skip this step if **google-services.json** has already been set up.
```

- Let's make [file names bold](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text) as per our writing style guide
- In this guide, above, we're using Firebase Console instead of Firebase Cloud Console. Either we use the former here too or change it above as per this change (to use the latter).
- We should be authoritative about placing the file so I think informing about placing it at the root for the project is fine in this case. We can also be explicit about it by adding the context, placing it anywhere else in the project directory is also fine.
- Let's move the not out from the first line and make it a callout.